helper-publish-gh-pages

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
Can I request changing the filename from `helper-publish-gh-pages` to `helper-publish-github-pages`? For the sake of a few characters, avoiding abbreviations can make names more immediately understandable for other users.

This was expressed as a [style rule](https://github.com/Financial-Times/dotcom-page-kit/blob/master/contribution.md#general-style) for Page Kit and I think it's a pretty good rule to apply generally.